deploy/001_ERC_4626_transformer.ts

```diff
@@ -17,9 +18,11 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
       values: [msig],
     },
     log: !process.env.TEST,
```

Comment:
None! I'll update for everything to be `env.isTesting()` 😊  - it kinda scares me that it will mean that all repos that import this deployment file will need to have `utils/env`.
What do you think? Should I just go for `!process.env.TEST` in this files?

Reply:
Yeah, let's go with `!process.env.TEST` then